src/iarc7_motion/iarc_tasks/track_roomba_task.py

```diff
@@ -146,9 +146,8 @@ def get_desired_command(self):
                 if not self._check_max_roomba_range():
                     return (TaskAborted(msg='The provided roomba is not found or not close enough to the quad'),)
 
-                roomba_x_velocity = self._roomba_odometry.twist.twist.linear.x
-                roomba_y_velocity = self._roomba_odometry.twist.twist.linear.y
-                roomba_velocity = math.sqrt(roomba_x_velocity**2 + roomba_y_velocity**2)
+                roomba_x_velocity = 0.0 #self._roomba_odometry.twist.twist.linear.x
```

Comment:
This needs to remain this way until the simple roomba estimator has better tracking of stopped versus moving states. I would like to merge this this way.

Reply:
I agree. Hopefully we will have something that can estimate the states soon.